src/condor_daemon_client/daemon.cpp

```diff
@@ -60,6 +60,7 @@ Daemon::common_init() {
 	Sock::set_timeout_multiplier( param_integer(buf, param_integer("TIMEOUT_MULTIPLIER", 0)) );
 	dprintf(D_DAEMONCORE, "*** TIMEOUT_MULTIPLIER :: %d\n", Sock::get_timeout_multiplier());
 	m_has_udp_command_port = true;
+	daemon_list_it = daemon_list.begin();
```

Comment:
I know this is not your doing, but it wasn't intuitively obvious to me on first reading of this code that `daemon_list` really is `collector_list`.  As long as you have this code in your head, should we change the name of this variable, and the related iterator?

Reply:
We should.